lib/perl/Genome/InstrumentData/AlignmentResult/Command/RecreatePerLaneBam.pm

```diff
@@ -32,6 +32,11 @@ class Genome::InstrumentData::AlignmentResult::Command::RecreatePerLaneBam {
             doc => 'Picard version to replace bam header',
             valid_values => [Genome::Model::Tools::Picard->available_picard_versions],
         },
+        bam_header => {
```

Comment:
It might be reasonable to make `bam_header` required and remove the heuristic for finding one via `_output_dir`.